sc2/client.py

```diff
@@ -118,7 +120,46 @@ async def leave(self):
         except (ProtocolError, ConnectionAlreadyClosed):
             if is_resign:
                 raise
+    
+    async def unload_unit(self, transporter_unit: Unit, cargo_unit: Unit = False):
+        """ 
+            Unloads single unit passed by cargo_unit or first unit in transporter if not cargo_unit passed 
+            transporter_unit includes all units, which can unload cargo: 
+            warp prism, medivac, bunker, command center, command center flying, planetary fortress, droppenlord, nydlus
+            
+            Usage:
+            self.client.unload_unit(transporter_unit, cargo_unit)
+            self.client.unload_unit(transporter_unit) # unloads first one
```

Comment:
Why would you prefer this function to be in the client.py instead of bot_ai.py?

Reply:
bot_ai.py would be my second choice
In my first comment I list why client.py is my choice, what are arguments in favor of bot_ai.py?